ci/test_conda.sh

```diff
@@ -47,8 +49,13 @@ EXITCODE=0
 trap "EXITCODE=1" ERR
 set +e
 
+if [ "$USE_PYNVJITLINK" == true ]; then
+    rapids-logger "Install pynvjitlink"
+    conda install -c rapidsai pynvjitlink
```

Comment:
I think you need to temporarily allow unbound locals for conda operations:
```suggestion
    set +u
    conda install -c rapidsai pynvjitlink
    set -u
```